background_scripts/main.js

```diff
@@ -169,6 +169,46 @@ const mkRepeatCommand = (command) => (function (request) {
   }
 });
 
+async function setZoom(direction, { count, tabId }) {
+  const zoomLevels = [
+    0.25,
+    0.33,
+    0.50,
+    0.75,
+    0.80,
+    0.90,
+    1.00,
+    1.10,
+    1.25,
+    1.50,
+    1.75,
+    2.00,
+    2.50,
+    3.00,
+    4.00,
+    5.00,
+  ];
+  const currentZoom = await chrome.tabs.getZoom(tabId);
+  let newZoom;
+  switch (direction) {
+    case "in":
+      // Round down to the nearest zoom index.
+      const floorIndex = zoomLevels.findIndex((level) => level > currentZoom) - 1;
```

Comment:
These lines are this way so that it handles zooming from between values correctly. For example, if we are at 225% and zoom in, we should go to 250%. Or if we zoom out, we should go to 200%. But, we also want, if we are at 200% and zoom out to go to 175% and in to go to 250%.
This implementation does that and seems reasonable, but if you have a more understandable way to achieve this, please suggest it.